src/components/nav.test.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import renderer from 'react-test-renderer';
+import Nav from './nav.js';
+
+it('renders home', () => {
+  const nav = renderer.create(
+    <Router>
+    <Nav />
+    </Router>,
+  ).toJSON();
+  expect(nav).toMatchSnapshot();
+});
```

Comment:
- [ ] The nav is another area where we also need to test user interaction since it contains links that we can use to navigate to different pages of the app. Please let us also include interactions tests for this component. Since it is the App component that loads the routes, tests for navigation can/should be written there.

### HINT:
* Import `userEvent` from '@testing-library/user-event'

### Example:

```JavaScript
// App.test.js file

import { render, screen } from '@testing-library/react';
import userEvent from '@testing-library/user-event';
import App from '../App';

describe('Navigation links leading to correct route in App', () => {
  beforeEach(() => {
    render(<App />);
  });

  test('Clicking on About link displays About page', () => {
    const aboutLink = screen.getByText('About');

    userEvent.click(aboutLink);
    const aboutText = screen.getByText('About Us!');

    expect(aboutText).toBeVisible();
  });
});
```

## REMINDER
Rember to add create the `App.test.js` file, add snapshot test for App component in it and then write user interactions tests as demonstrated above. All the best.